dDatabase/CDClientDatabase.h

```diff
@@ -40,4 +40,10 @@ namespace CDClientDatabase {
      */
     CppSQLite3Query ExecuteQuery(const std::string& query);
     
+    //! Queries the CDClient and parses arguments
+    /*!
+      \param query The query with formatted arguments
+      \return prepared SQLite Statement
+    */
+    CppSQLite3Statement CreatePreppedStmt(const std::string& query);
```

Comment:
I'd personally prefer `CreatePreparedStmt`

Reply:
My thinking was to use the same name as the corresponding mysql function
https://github.com/DarkflameUniverse/DarkflameServer/blob/3de393250340ad27aa2dcac39afe9ed93c114959/dDatabase/Database.h#L27

Do you want me to change this?

Reply:
No, that's a pretty good reason to keep it like this